src/world.ts

```diff
@@ -521,16 +553,4 @@ export class World {
     this.robots = [];
     this.app.destroy(true, true);
   }
-
-  private initAudioContext() {
-    window.audioContext = new AudioContext();
-    const listener = window.audioContext.listener;
```

Comment:
We are no longer using spatialized nodes like `PannerNode` (instead we control the pan directly using `StereoPannerNode`), so we don't actually need to do anything with the listener. Just removing this part.